.github/workflows/docker-build.yaml

```diff
@@ -1,58 +1,58 @@
-name: Docker images building and pushing
-on:
-  push:
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+"
-      - "v[0-9]+.[0-9]+.[0-9]+-*"
-
-jobs:
-  docker:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
-      - name: Docker meta for Testkube operator
-        id: meta_testkube_operator
-        uses: docker/metadata-action@v3
-        with:
-          images: kubeshop/testkube-operator
-          tags: |
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=sha
-
-      - name: Build
-        uses: docker/bake-action@v1
-        with:
-          files: |
-            docker-bake.hcl
-            ${{ steps.meta_testkube_operator.outputs.bake-file }}
-          targets: build
-          push: true
-
-      - name: Docker meta for Testkube certificate
-        id: meta_testkube_certificate
-        uses: docker/metadata-action@v3
-        with:
-          images: kubeshop/testkube-certificate
-          tags: |
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=sha
+#name: Docker images building and pushing
```

Comment:
do we need commented code?

Reply:
I will remove it after we decide to go with the solution. The main point is in the description.